web/src/stats/stats.ts

```diff
@@ -24,14 +27,15 @@ type PlotTrace = {
     trace: Partial<Plotly.PlotData>;
 };
 
-// TODO: Merge this with DataByEveryoneMe.
 type DataByEveryoneUser<T> = {
     everyone: T;
     user: T;
 };
 
-// TODO: Convert DataByEveryoneUser to this.
-type DataByEveryoneMe<T> = {everyone: T; me: T};
+type DataByEveryoneMe<T> = {
+    everyone: T;
+    me: T;
+};
```

Comment:
Looks like this change and the one above belong to a previous commit.

Reply:
Fixed!